hardware/Rtl433.cpp

```diff
@@ -232,11 +233,12 @@ void CRtl433::Do_Work()
 				try {
 					if (!data["humidity"].empty())
 					{
-						humidity = boost::lexical_cast<int>(data["humidity"]);
+						humidity = static_cast<int>(boost::math::round(boost::lexical_cast<float>(data["humidity"])));
```

Comment:
Please remove the boost dependency... have a small search for the 'round' function used by various other classes (#define)

Reply:
Sure, I'll take care of that, thanks.